lib/services/northBound/contextServer-NGSI-LD.js

```diff
@@ -290,7 +290,7 @@ function handleQueryNgsiLD(req, res, next) {
 
```

Comment:
The fix in this PR should be described in CHANGES_NEXT_RELEASE file

Reply:
Fixed f8380a7